### PR TITLE
.github: add release template for RCs from master branch

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release_template_rc_master.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+This kicks off the release train that leads us towards vX.Y final version in the coming weeks. There are [vX.Y.Z-rcW OSS docs](https://docs.cilium.io/en/vX.Y.Z-rcW) available if you want to pull this version & try it out.
+```
+Text template for the next RCs:
+```
+*Announcement* :tada: :tada:
+
+:cilium-new: *Cilium release candidate vX.Y.Z-rcW has been released:*
+https://github.com/cilium/cilium/releases/tag/vX.Y.Z-rcW
+
+Thank you for the bugs reported found in the previous RC. There are [vX.Y.Z-rcW OSS docs](https://docs.cilium.io/en/vX.Y.Z-rcW) available if you want to pull this version & try it out.
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/release_template_rc_master.md:83]: nit, RCs are partly about bugs, but also about docs improvements, other feature freeze exception PRs, etc. Broadly, contributions.

```suggestion
Thank you for testing and contributing to the previous RC. There are [vX.Y.Z-rcW OSS docs](https://docs.cilium.io/en/vX.Y.Z-rcW) available if you want to pull this version & try it out.
```

File: .github/ISSUE_TEMPLATE/release_template_rc_master.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+## Post-release
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/release_template_rc_master.md:86]: Are there any post-release steps?

**Review by joestringer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/release_template_rc_master.md (hunk 2 above)
Comment (aanm) [reply .github/ISSUE_TEMPLATE/release_template_rc_master.md:86]: No, somehow I forgot to remove this.

File: .github/ISSUE_TEMPLATE/release_template_rc_master.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- [ ] Make sure docker is running.
+- [ ] Export a `GITHUB_TOKEN` that has access to the repository.
+- [ ] Check that there are no [release blockers] for the targeted release version
+- [ ] Consider building new [cilium-runtime images] and bumping the base image
+      versions on this branch:
+  - Cilium v1.10 or later:
+     Modify the `FORCE_BUILD` environment value in the `images/runtime/Dockerfile` to force a rebuild.
+     [Instructions](https://docs.cilium.io/en/latest/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images)
+  - Cilium v1.7 to v1.9:
+     [Re-trigger a build in quay.io](https://docs.cilium.io/en/v1.9/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images)
```
[the remark this answers is not in the document]
Comment (aanm) [reply .github/ISSUE_TEMPLATE/release_template_rc_master.md:25]: Yes, this can be removed because this is RC from master branch.

**Review by joestringer — APPROVED** [review]

**Pull request merged**